content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -151,10 +152,6 @@ different Kubernetes components.
 | `ProcMountType` | `false` | Alpha | 1.12 | |
 | `QOSReserved` | `false` | Alpha | 1.11 | |
 | `RemainingItemCount` | `false` | Alpha | 1.15 | |
-| `RemoveSelfLink` | `false` | Alpha | 1.16 | 1.19 |
-| `RemoveSelfLink` | `true` | Beta | 1.20 | |
```

Comment:
Why is RemoveSelfLink removed here? The PR description does not explain this amendment.

Reply:
oops, done